### PR TITLE
FCCEUD-6 Create a process for accepting feedback and updating the end-user documentation

### PR DESCRIPTION
**Pull request by stoobie**

### CODE REVIEW

**Comment by stoobie** [comment]
@redhataccess/eud-tech-review @dvagnero  Please review for technical accuracy. This is about using Jira to file doc requests.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ include::../attributes.adoc[]
 ifdef::context[:parent-context: {context}]
 
 [id="assembly-help_{context}"]
-= Pantheon Help Page
+= {ProductName} User Guide
```
Comment (dvagnero) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc:13]: We're not publishing this document in Pantheon 2, right? I'm not 100 % sure atm. If yes, then we cannot have attributes in the titles.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+_module-type: PROCEDURE
+[id="proc_submitting-documentation-feedback_{context}"]
+= Submitting documentation feedback
+
+[role="_abstract"]
+If you encounter any documentation bugs or have any requests for additions or changes to the {ProductName} end-user documentation, you can open a Jira ticket to create a request for the documentation team.
+
+Do not submit documentation feedback to submit a bug or functionality request for Pantheon.
```
Comment (dvagnero) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc:8]:
```suggestion
Do not submit documentation feedback to report a bug or to create a functionality request for Pantheon.
```

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+.Procedure
+
+. Log in to link:https://projects.engineering.redhat.com/projects/FCCEUD/issues[the Jira project for the Pantheon 2 end-user documentation (FCCEUD)]. This is the internal Red Hat instance of Jira, so you need to be connected to the Red Hat network in an office or using the VPN.
+. Click *Create*.
+. Fill out the form as follows:
++
```
Comment (dvagnero) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc:15]: You can keep it as it is or use substeps. The mod-docs templates use substeps

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+Project:: *FCC - Pantheon V2 End-User Documentation* (FCCEUD)
+Issue Type:: *Bug* or *Task*
+Summary:: This is the title of the ticket. Enter *`[FEEDBACK]`* Followed by a short description of the issue. For example: `[FEEDBACK] Broken link in module "Module types"`
+Description:: Enter information that fully explains the feedback. Include as much detail as you can, including any questions you would like answered.
```
Comment (dvagnero) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc:19]: I'd add the following line:
`. Click *Create*.`

**Comment by benradey** [comment]
Looks good to me, tech review complete. I see @dvagnero already applied the label. :)

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc (hunk 1 above)
Comment (stoobie) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc:13]: @dvagnero We are *not* publishing the EUD in Pantheon 2 (at the moment, at least). I'm going to leave this as-is.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc (hunk 3 above)
Comment (stoobie) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc:15]: @dvagnero The type of list to use should be guided by the type of info. Steps or substeps indicate a necessary order to follow. In this case, we're just describing a set of fields to fill out. It doesn't really matter what order, and I think that a two column list (i.e. definition list) is more semantically correct and easier to parse visually.

There are a lot of Asciidoc features that the templates don't include.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc (hunk 4 above)
Comment (stoobie) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc:19]: Done

**Review by dvagnero — APPROVED** [review]

**Comment by stoobie** [comment]
@dvagnero I just made one more minor change. Can you please take a look and approve?

**Review by dvagnero — APPROVED** [review]

**Comment by stoobie** [comment]
@redhataccess/pantheon-developers This is ready for merging.

**Review by benradey — APPROVED** [review]

**Pull request merged**